installer/lib/phx_new/generator.ex

```diff
@@ -349,12 +349,14 @@ defmodule Phx.New.Generator do
           environment variable DATABASE_URL is missing.
           For example: ecto://USER:PASS@HOST/DATABASE
           \"""
+
+      ipv6? = System.get_env("ECTO_IPV6") == "true"
```

Comment:
Usually we don't check for true specifically, because it is common to set it to 1 or similar? We could simply set if the variable is set, but that will also enable it if it is empty... which might be ok? DATABASE_URL would be used even for empty. So maybe something like this:

```suggestion
      maybe_ipv6 = if System.get_env("ECTO_IPV6"), do: [:inet6], else: []
```

And below you do `socket_options: maybe_ipv6`?